draft-ietf-regext-rdap-geofeed.md

```diff
@@ -32,107 +32,208 @@ email = "tomh@apnic.net"
 
 {mainmatter}
 
 # Introduction
 
-[@!RFC8805] and [@?I-D.ymbk-opsawg-9092-update] (obsoletes [@!RFC9092]) detail
-the IP geolocation feed (in short, geofeed) concept. This document specifies
-how the geofeed data can be accessed through RDAP. It defines a new RDAP
-extension `geofeedv1` for including a geofeed file URL in an IP Network object.
+[@!RFC8805] and [@?I-D.ymbk-opsawg-9092-update] (obsoletes [@!RFC9092]) detail the IP geolocation feed (in short,
+geofeed) concept. This document specifies how the geofeed data can be accessed through RDAP. It defines a new RDAP
+extension "geofeed1" for including a geofeed file URL in an IP Network object.
 
 ## Requirements Language
 
-The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
-"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
-document are to be interpreted as described in BCP 14 [@!RFC2119] [@!RFC8174]
-when, and only when, they appear in all capitals, as shown here.
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD","SHOULD NOT", "RECOMMENDED",
+"NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14
+[@!RFC2119] [@!RFC8174] when, and only when, they appear in all capitals, as shown here.
 
 # Specification
 
 ## Extension
 
-A new RDAP extension `geofeedv1` is defined for accessing the geofeed data
-through RDAP. It extends the IP Network object class [@!RFC9083] to include
-a new `geofeed` member.
+A new RDAP extension "geofeed1" is defined for accessing the geofeed data through RDAP. It updates the IP Network object
+class definition ([@!RFC9083, section 5.4]) to include a new link object for the geofeed file URL in its "links" array
+([@!RFC9083, section 4.2]).
 
-An RDAP server conforming to this specification MUST include the `geofeedv1`
-extension string in the `rdapConformance` array for the IP Network lookup and
-search responses, as well as in the help response.
+An RDAP server conforming to this specification MUST include the "geofeed1" extension string in the "rdapConformance"
+array for the IP Network lookup and search responses, as well as in the help response.
 
-## Geofeed Member
+## Geofeed Link
 
-The IP Network object class is extended to include a new `geofeed` member, as
-specified below:
+The IP Network object class ([@!RFC9083, section 5.4]) MAY include a link object for the geofeed file URL in its "links"
+array, with the following REQUIRED JSON members:
 
-geofeed — a string specifying the HTTPS URL of the geofeed file for an IP
-          network [@!RFC9092]
+* "value" -- The "value" JSON value is the context URI and set to an IP Network lookup URL ([@!RFC9082, section 3.1.1]).
+* "rel" -- The "rel" JSON value is the link relation type and set to the "geofeed" string. The "geofeed" link relation
+type is new and will be registered in the IANA Link Relations Registry (see the "Link Relations Registry" section).
```

Comment:
To stay in sync with the RFC 9092 bis, recommend registering a more specific rel in "geofeed" instead of "geo". Thoughts?

Reply:
Since I was thinking more of tightly aligning with the RFC 9092 bis vis-a-vis the terms we pick for the new web link, I was focusing more on "They SHOULD be appropriate to the specificity of the relation type; that is, if the semantics are highly specific to a particular application, the name should reflect that, so that more general names are available for less-specific use." in the same section. That said, I think we'll go with "geo", in lieu of "geofeed".